src/standalone/Soil/soil_albedo.jl

```diff
@@ -0,0 +1,221 @@
+export update_albedo!, CLMTwoBandSoilAlbedo, ConstantTwoBandSoilAlbedo
+
+abstract type AbstractSoilAlbedoParameterization end
+
+"""
+    ConstantTwoBandSoilAlbedo{
+        SF <: Union{AbstractFloat, ClimaCore.Fields.Field},
+    } <: AbstractSoilAlbedoParameterization
+
+A parameterization for soil albedo: the soil albedo is 
+defined in two bands (PAR and NIR), can spatially vary or
+be set to scalar, but is assumed not to vary in time (or
+with water content).
+"""
+struct ConstantTwoBandSoilAlbedo{
+    SF <: Union{AbstractFloat, ClimaCore.Fields.Field},
+} <: AbstractSoilAlbedoParameterization
+    "Soil PAR Albedo"
+    PAR_albedo::SF
+    "Soil NIR Albedo"
+    NIR_albedo::SF
+end
+
+function ConstantTwoBandSoilAlbedo{FT}(;
+    PAR_albedo::FT = FT(0.2),
+    NIR_albedo::FT = FT(0.4),
+) where {FT}
+    ConstantTwoBandSoilAlbedo{FT}(PAR_albedo, NIR_albedo)
+end
+
+
+"""
+    update_albedo!(bc::AtmosDrivenFluxBC, albedo::ConstantTwoBandSoilAlbedo, p, soil_domain, model_parameters)
+
+Updates PAR and NIR albedo using the temporally-constant parameters 
+provided in `albedo`; these values may be spatially varying.
+
+For the temporally-constant albedo model, there is no need to update 
+the values each step, or have an allocated spot in the cache for them. This
+can be optimized in the future.
+"""
+function update_albedo!(
+    bc::AtmosDrivenFluxBC,
+    albedo::ConstantTwoBandSoilAlbedo,
+    p,
+    soil_domain,
+    model_parameters,
+)
+    @. p.soil.PAR_albedo = albedo.PAR_albedo
+    @. p.soil.NIR_albedo = albedo.NIR_albedo
+end
+
+"""
+     CLMTwoBandSoilAlbedo{
+        FT <: AbstractFloat,
+        SF <: Union{FT, ClimaCore.Fields.Field},
+    } <: AbstractSoilAlbedoParameterization
+
+A parameterization for soil albedo: the soil albedo is 
+defined in two bands (PAR and NIR), and can spatially vary or
+be set to scalar. However, it varies temporally due to a
+dependence on soil water content,
+θ_sfc, according to CLM:
+
+α = min(α_wet + Δ(θ_sfc), α_dry), where
+Δ(θ_sfc) = max(θ_int - dαdθ*θ_sfc,0),
+where θ_int = 0.11 and dαdθ = 0.4.
```

Comment:
In the paper Braghiere et al. (2023b) we validated this scheme vs. the updated one (weighted averages), and we showed a better agreement of the updated one. I wonder if we shouldn't just use the updated one instead: 

https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2023AV000910

Reply:
Happy to go with what you think is best. My thought was that the parameters we read in from CLM for dry and wet albedos were calibrated with this combination of them, so if we use the CLM parameters we should use their linear combination also. If we use the different linear combination, do we need to recalibrate the soil wet and dry albedo values by color?

Reply:
Thanks, that makes sense. Just to clarify: in our 2023b paper we used theta directly from SMAP because using S would require knowing both theta_r and theta_sat, which introduces uncertainty due to spatial variability in soil properties. One advantage of the theta-based scheme was that it was fully observation-driven. 

But I agree. if we're working in a model that defines theta_r and theta_sat, then using S is more consistent with the physics.